fdbserver/DataDistribution.actor.cpp

```diff
@@ -5252,12 +5260,19 @@ ACTOR Future<Void> dataDistribution(Reference<DataDistributorData> self,
 			primaryTeamCollection = Reference<DDTeamCollection>();
 			remoteTeamCollection = Reference<DDTeamCollection>();
 			wait(shards.clearAsync());
-			if (err.code() != error_code_movekeys_conflict)
-				throw err;
-			bool ddEnabled = wait(isDataDistributionEnabled(cx, ddEnabledState));
-			TraceEvent("DataDistributionMoveKeysConflict").detail("DataDistributionEnabled", ddEnabled).error(err);
-			if (ddEnabled)
-				throw err;
+			TraceEvent("DataDistributorTeamCollectionsDestroyed").error(err);
```

Comment:
This trace seems redundant because a similar trace already exists on one of the lines above

Reply:
I added it so that we would know when clearAsync completes